packages/frontend-shared/cypress/e2e/support/e2eProjectDirs.ts

```diff
@@ -13,6 +13,8 @@ export const e2eProjectDirs = [
   'config-with-short-timeout',
   'config-with-ts',
   'cookies',
+  'create-react-app-configured',
```

Comment:
This is the best way I could find to verify the generated `cypress.config.js` works. Each project has two variants:

1. `<project>-configured`
2. `<project>-unconfigured`


The `-configured` variant is ready to run - it has the valid `cypress.config.js`. We verify it is valid by actually running the project via a system test: https://github.com/cypress-io/cypress/pull/20306/files#diff-9a09b374d404587d6bc455d2e1989ca5637a2e8cabbca5be488050a71ccb94f7. If the config is invalid, the system test will fail.

The `-unconfigured` variant is used by the launchpad e2e test: https://github.com/cypress-io/cypress/pull/20306/files#diff-757686eab70fe8cb1974b1e8d3fcdbbd2fde3fde61420e9708efcbc740e3fa47R11-R19. We select an unconfigured project, configure it with launchpad, and finally verify the generated `cypress.config.js` we create is identical to the one in `-configured`, which we know works since we have a system test for it.

A bit of work involved, but this gives me confidence we are generating a config file that is proven to work correctly.